wings/1.0/configuration.md

```diff
@@ -4,23 +4,24 @@
 
 ::: warning
 These are advanced configurations for Wings. You risk breaking Wings and making containers unusable if
-you configure something incorrectly. Proceed only if you know what each configuration value does.
+you misconfigure something. Proceed only if you know what each configuration value does.
 :::
 
+You must apply all changes to your Wings `config.yml` file located at `/etc/pterodactyl`. Verify your config file using [Yaml Lint](http://www.yamllint.com/) should you receive errors related to YAML parsing.
```

Comment:
"...wings config file located at `/etc/pterodactyl/config.yml`" would be better here, so you can copy paste imo

Reply:
I've followed the pattern of the wings installation documentation where the file and location are specified separately